services/core-web/webpack.config.ts

```diff
@@ -21,6 +21,7 @@ const BUILD_DIR = process.env.BUILD_DIR || "build";
 const smp = new SpeedMeasurePlugin({
   disable: !process.env.MEASURE_SPEED,
 });
+const ReactRefreshPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
```

Comment:
Snuck in an improvement for hot reloading in core using react-refresh instead of the old hmr. Seems to be working a fair bit better, the amount of times I've had to refresh the page to view changes over the past week has been greatly reduced